docs/compilers/CSharp/Compiler Breaking Changes - DotNet 7.md

```diff
@@ -1,5 +1,22 @@
 # This document lists known breaking changes in Roslyn after .NET 6 all the way to .NET 7.
 
+## Type tests for `ref` structs ar not supported.
+
+***Introduced in Visual Studio 2022 version 17.4***
+
+When a `ref` struct type is used in an 'is' or 'as' operator, in some scenarios compiler was previously reporting
+an erroneous warning about the type test always failing at runtime, omitting the actual type check, and leading to
+an incorrect behavior. When an incorrect behavior at execution time was possible, compiler will now produce an error instead.
```

Comment:
`an` should be removed in both places on this line.

Reply:
> ```an``` should be removed in both places on this line.

Could you clarify please? There are three places where ```an``` is used on this line.

Reply:
Sorry, yes.
```suggestion
incorrect behavior. When incorrect behavior at execution time was possible, compiler will now produce an error instead.
```